ecommerce/CartController.php

```diff
@@ -121,64 +136,58 @@ public static function get_page_id() {
 	}
 
 	/**
-	 * Retrieve cart data by cart ID and respond with JSON.
+	 * Get cart items
 	 *
-	 * This function retrieves the cart ID from the POST request, validates it,
-	 * fetches the corresponding cart data using the CartModel class, and returns
-	 * a JSON response with the cart data or an error message.
+	 * @since 3.0.0
 	 *
-	 * If the cart ID is not provided, it responds with a "Bad Request" status.
-	 * If the cart is not found, it responds with a "Not Found" status.
-	 * Otherwise, it responds with the cart data and a success message.
+	 * @return array
+	 */
+	public function get_cart_items() {
+		$user_id = tutils()->get_user_id();
+		return $this->model->get_cart_items( $user_id );
+	}
+
+	/**
+	 * Add course to cart
 	 *
 	 * @since 3.0.0
 	 *
-	 * @return void
+	 * @return void JSON response
 	 */
-	public function get_cart_by_id() {
+	public function tutor_add_course_to_cart() {
 		if ( ! tutor_utils()->is_nonce_verified() ) {
-			$this->json_response( tutor_utils()->error_message( 'nonce' ), null, HttpHelper::STATUS_BAD_REQUEST );
+			wp_send_json_error( tutor_utils()->error_message( 'nonce' ) );
```

Comment:
Here we are sending message directly in the body which is like: 
`{success: false, data: 'msg'}`
But on the JS side, we are expecting `message` key inside `data`